src/app/services/property-detail/property-detail.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from "@angular/core";
+import { HttpHeaders } from "@angular/common/http";
+import { Observable } from "rxjs";
+import { HttpService } from "../http.service";
+import { LocalStorageService } from '../local-storage.service'
+
```

Comment:
@dorothyas I can see where we are using `HttpHeaders` and `LocalStorage` we are importing here. Please remove any unused import.